functions/system.bash

```diff
@@ -26,13 +26,13 @@ needed_packages() {
   # Install apt-transport-https - update packages through https repository
   # Install bc + sysstat - needed for FireMotD
   # Install avahi-daemon - hostname based discovery on local networks
-  # Install python/python-pip - for python packages
+  # Install python/python3-pip - for python packages
   echo -n "$(timestamp) [openHABian] Installing additional needed packages... "
-  if cond_redirect apt-get -y install apt-transport-https bc sysstat avahi-daemon python python-pip avahi-autoipd fontconfig; then echo "OK"; else echo "FAILED"; exit 1; fi
+  if cond_redirect apt-get -y install apt-transport-https bc sysstat avahi-daemon python3 python3-pip avahi-autoipd fontconfig; then echo "OK"; else echo "FAILED"; exit 1; fi
 
   if is_pithree || is_pithreeplus || is_pizerow; then
     echo -n "$(timestamp) [openHABian] Installing additional bluetooth packages... "
-    if cond_redirect apt-get -y install bluez python-bluez python-dev libbluetooth-dev raspberrypi-sys-mods pi-bluetooth; then echo "OK"; else echo "FAILED"; exit 1; fi
+    if cond_redirect apt-get -y install bluez python3-bluez python3-dev libbluetooth-dev raspberrypi-sys-mods pi-bluetooth; then echo "OK"; else echo "FAILED"; exit 1; fi
```

Comment:
So effectively this was the only place (the rest is just used in Travis testing).

However, IIRC Raspbian still has python2 as default what "python" links to, so you actually have to replace all occurrences of "python" and "pip" by "python3"/"pip3", don't you ?

Reply:
I think I got everything replaced. Let's see what the tests do and then also try a fresh install....

Are you aware of any dependencies of OH2.5 towards Python version 2?

Reply:
no

Reply:
@mstormi any idea why starting openhab fails in Travis builds for this PR? On my rpi it works.
https://travis-ci.org/github/holgerfriedrich/openhabian/jobs/685007366
Do you know more about the systemd to replacement in those docker images?